include/deal.II/lac/sparse_matrix_tools.h

```diff
@@ -262,22 +266,13 @@ namespace SparseMatrixTools
 
           const unsigned int row_length = sparsity_pattern.row_length(row);
 
-
-          std::pair<types::global_dof_index,
-                    std::vector<std::pair<types::global_dof_index, Number>>>
-            buffer;
           buffer.first = row;
```

Comment:
It looks like `buffer.first` isn't used any more - can you get rid of it?

Reply:
It is: https://github.com/dealii/dealii/blob/c517485f61e10901304462e0cf1a55626357992f/include/deal.II/lac/sparse_matrix_tools.h#L276

Reply:
ah there it is!